js/RecentDonations.js

```diff
@@ -0,0 +1,196 @@
+(function() {
+var app = angular.module('RecentDonations', []);
+app.controller('RecentDonationsControl', function($scope) {
+
+    var urlVars = GLOBALS.getUrlVars();
+    var count = Number(urlVars['count']);
+    var uid = urlVars['id'];
+    var isTeam = urlVars['team'];
+
+    if (count === undefined) { count = 5 }
```

Comment:
nitpicky, but: this should go right after the count definition line so we see right away what it's value will be